packages/headless/src/case-assist.ts

```diff
@@ -0,0 +1,3 @@
+export function hello() {
```

Comment:
Also, I think it would be nicer if we had: 

```
packages/headless/src/case-assist/index.ts
```

Reply:
I will empty the file and keep just a comment for git.

For creating a `case-assist` directory, I feel this divides `case-assist` functionality into it's own branch of the tree, which I don't think we should do.

Currently it looks like this:

![headless_directory_structure](https://user-images.githubusercontent.com/5565841/115281442-05e2c700-a117-11eb-84dd-8f794514c288.PNG)


I think we should keep controllers together, features together, and group functionality under those concepts, rather than by sub-package. Some controllers will be used by multiple packages after all.

```
controllers/
    controller1
    controller2
```

If the goal behind the directory is to have the word `index` in the file name though, we could call it:

```
case-assist.index.ts
```

Reply:
Edit: readding a dummy export because rollup throws an error otherwise.